doc/nrf/protocols/bt/bt_mesh/overview/coexistence.rst

```diff
@@ -0,0 +1,10 @@
+.. ug_bt_mesh_overview_coexistence:
+
+Bluetooth LE and Bluetooth Mesh coexistence
+###########################################
+
+Bluetooth Mesh uses the preset local identity value.
+For more information about the advertisement identity, see :ref:`zephyr:bluetooth_mesh_adv_identity`.
+
+Bluetooth Mesh sample :ref:`bluetooth_ble_peripheral_lbs_coex` demonstrates how to perform correct local identity allocation for Bluetooth LE purposes to avoid identity conflict with Bluetooth Mesh.
```

Comment:
```suggestion
The :ref:`bluetooth_ble_peripheral_lbs_coex` sample demonstrates how to perform correct local identity allocation for Bluetooth LE purposes to avoid identity conflict with Bluetooth Mesh.
```
The sample name includes Bluetooth Mesh.